### ISSUE TITLE
The last blank is trimed，so how to fix this error

### ISSUE THREAD

**Issue by gclinux** [apla/node-clickhouse#25]
error:
Error: Unknown format CSV1550220733

Code

> let statement =  'INSERT INTO testTable FORMAT CSV ';
>         let csvStream = fs.createReadStream (‘csv.csv’);
>         let clickhouseStream = ch.query (statement, {inputFormat: 'CSV'});
>         csvStream.pipe (clickhouseStream);
>         clickhouseStream.on('end',function(){
>             ac(file_name+'->success');
>         });
>         clickhouseStream.on ('error', function (err) {
>            console.log(err);
>            ac(file_name+'->false');
>         });
> 

csv file

`1550220733,22222,33333`

**Comment by gclinux** [comment]
I have fixed。
let clickhouseStream = ch.query (statement, {inputFormat: 'CSV'});
**clickhouseStream.write('\n');**
csvStream.pipe (clickhouseStream);

**Comment by nezed** [comment]
There is a bug with while serializing the body of request — data aren't separated from query with space or newline.
Also, the `inputFormat` option from readme aren't supported.

I'll prepare fix for this.

For now better way is to use `format` options instead of `FORMAT` statement.
```js
let clickhouseStream = ch.query ('INSERT INTO testTable', {format: 'CSV'});
```

**Comment by nezed** [comment]
@gclinux  Fixed in `@apla/clickhouse@1.6.1`

https://github.com/apla/node-clickhouse/releases/tag/v1.6.1
